apple/internal/ios_rules.bzl

```
@@ -641,6 +651,8 @@ def _ios_app_clip_impl(ctx):
         executable_name = executable_name,
         platform_prerequisites = platform_prerequisites,
         predeclared_outputs = predeclared_outputs,
+        rule_descriptor = rule_descriptor,
+        label_name = label.name,
```

It would be nice to keep these alphabetical (here and the other places you added it):

```suggestion
        label_name = label.name,
        platform_prerequisites = platform_prerequisites,
        predeclared_outputs = predeclared_outputs,
        rule_descriptor = rule_descriptor,
```

---

Oh yes, makes sense, updated!